.vsts-ci/sshremoting-tests.yml

```diff
@@ -0,0 +1,97 @@
+name: PR-$(System.PullRequest.PullRequestNumber)-$(Date:yyyyMMdd)$(Rev:.rr)
```

Comment:
@PaulHigin, your last commit had 1 failures in `PowerShell-CI-linux`
Test-Connection.Error occurred in Describe block
```powershell
Exception calling "GetHostEntry" with "1" argument(s): "Name or service not known"
At /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1:15 char:9
```